model_analyzer/result/result_manager.py

```diff
@@ -401,6 +405,7 @@ def _tabulate_measurements(self, result):
         instance_group = result.model_config().instance_group_string()
         dynamic_batching = result.model_config().dynamic_batching_string()
         cpu_only = result.model_config().cpu_only()
+        backend_parameters = result.model_config()._model_config.parameters
```

Comment:
confirm this works if parameters doesn't exist in the protobuf object

Reply:
I think this is tested in the CI because the models in the CI do not have backend parameters.

Reply:
i confirmed it just returns an empty `google.protobuf.pyext._message.MessageMapContainer` (`{}`), which is fine